omnigibson/scene_graphs/graph_exporter.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+import numpy as np
+
+from igibson.scene_graphs.enum_obj_states import BinaryStatesEnum, UnaryStatesEnum
+from igibson.scene_graphs.graph_builder import SceneGraphBuilder
+
+
+class SceneGraphExporter(SceneGraphBuilder):
+    def __init__(self, h5py_file, num_frames_to_save=None, full_obs=False, **kwargs):
+        """
+        @param h5py_file: file to store exported scene graph
+        @param num_frames_to_save: total number of equidistant frames to save. None if you want to set all frames.
+        """
+        if num_frames_to_save is None:
+            assert full_obs, "You can only use full observability mode if you're subsampling frames."
+
+        super(SceneGraphExporter, self).__init__(only_true=True, full_obs=full_obs, **kwargs)
+        self.h5py_file = h5py_file
+
+        self.num_frames_to_save = num_frames_to_save
+        self.frame_idxes_to_save = None
+
+    def start(self, activity, log_reader):
+        super(SceneGraphExporter, self).start(activity, log_reader)
+
+        scene = activity.simulator.scene
+        objs = set(scene.objects()) | set(activity.object_scope.values())
+        self.num_obj = len(objs)
+        self.dim = 34  # 34: dimension of all relevant information
```

Comment:
why is this hardcoded? what does 34 represent?

Reply:
Removed this file for now.